### PR TITLE
Clean-up HTML

### PR DESCRIPTION
**Pull request by ahohnsen**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,117 +1,158 @@
...
+              sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit
+              amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor
+              invidunt ut labore et dolore magna aliquyam erat, sed diam
+              voluptua. At vero eos et accusam et justo duo dolores et ea rebum.
+              Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum
+              dolor sit amet.
+            </p>
+          </section>
+          <section class="section-box alignment">
+            <dl>
```
Comment (lalueders) [inline index.html:94]: Nice! Love the `<dl>`. Changed my code the same way!

**Review by lalueders — APPROVED** [review]
Looks good to me!

File: index.html (hunk 1 above)
Comment (ahohnsen) [reply index.html:94]: Yes, so much cleaner, hehe. Thank you!

**Review by FaGru — APPROVED** [review]
i can't see any problems 👍🏻
but maybe Lasse 😉

**Pull request merged**